src/extensions/starter_dashlet/Tools.tsx

```diff
@@ -445,17 +383,24 @@ class Starter extends ComponentEx<IStarterProps, IWelcomeScreenState> {
   }
 
   private makePrimary = (starter: StarterInfo) => {
```

Comment:
this name doesn't seem accurate any more as the function now seems to act more like a toggle, right?